File: src/main/java/phoenix/client/render/entity/TalpaRenderer.java
package phoenix.client.render.entity;

import net.minecraft.client.renderer.entity.EntityRendererManager;
import net.minecraft.client.renderer.entity.MobRenderer;
import net.minecraft.util.ResourceLocation;
import phoenix.Phoenix;
import phoenix.client.models.entity.TalpaModel;
import phoenix.enity.TalpaEntity;

public class TalpaRenderer extends MobRenderer<TalpaEntity, TalpaModel>
{
    public TalpaRenderer(EntityRendererManager renderManager)
    {
        super(renderManager, new TalpaModel(), 0.5F);
    }

    @Override
    public ResourceLocation getEntityTexture(TalpaEntity entity)
    {
        return entity.isChild() ? new ResourceLocation(Phoenix.MOD_ID, "textures/entity/talpa_child.png") :
                                  new ResourceLocation(Phoenix.MOD_ID, "textures/entity/talpa.png");
    }
}

File: src/main/kotlin/phoenix/client/render/entity/TalpaRenderer.kt
package phoenix.client.render.entity

import net.minecraft.client.renderer.entity.EntityRendererManager
import net.minecraft.client.renderer.entity.MobRenderer
import net.minecraft.util.ResourceLocation
import phoenix.MOD_ID
import phoenix.Phoenix
import phoenix.client.models.entity.TalpaModel
import phoenix.enity.TalpaEntity

class TalpaRenderer(renderManager: EntityRendererManager) : MobRenderer<TalpaEntity, TalpaModel>(renderManager, TalpaModel(), 0.5f)
{
    override fun getEntityTexture(entity: TalpaEntity) = ResourceLocation(MOD_ID, "textures/entity/" + if (entity.isChild) "talpa_child.png" else "talpa.png")
}
